cdatransform/extract/idc.py

```diff
@@ -0,0 +1,157 @@
+import argparse
+from google.cloud import bigquery, storage
+from google.oauth2 import service_account
+from cdatransform.lib import get_case_ids
+
+idc_fields = [
+    "PatientID",
+    "BodyPartExamined",
+    "StudyDescription",
+    "Modality",
+    "collection_id",
+    "crdc_study_uuid",
+    "crdc_series_uuid",
+    "crdc_instance_uuid",
+    "Program",
+    "tcia_tumorLocation",
+    "source_DOI",
+    "AnatomicRegionSequence",
+]
+
+
+class IDC:
+    def __init__(
+        self,
+        gsa_key='../../GCS-service-account-key.json',
+        cases_file=None,
+        case=None,
+        dest_table_id='gdc-bq-sample.idc_test.dicom_pivot_wave1',
+        dest_bucket='gdc-bq-sample-bucket',
+        dest_bucket_file_name='druth/idc-extract.jsonl.gz',
```

Comment:
I know we can override these but is there any risk to predefine the values so specifically,  is the bucket file name pointing to your identifier risky because it will overwritten by default.